test/distributed/test_dynamo_distributed.py

```diff
@@ -724,6 +724,7 @@ def opt_fn(inputs):
 
 
     @patch.object(config, "optimize_ddp", True)
+    @patch.object(config, "optimize_ddp_lazy_compile", True)
```

Comment:
wonder is there a way to check that it does produce the error message that you added in this PR?